example/lib/main.dart

```diff
@@ -5,69 +5,96 @@ void main() {
   runApp(MyApp());
 }
 
-class MyApp extends StatefulWidget {
+class MyApp extends StatelessWidget {
   @override
-  _MyAppState createState() => _MyAppState();
+  Widget build(BuildContext context) {
+    return MaterialApp(home: MyPage());
+  }
+}
+
+class MyPage extends StatefulWidget {
+  @override
+  _MyPageState createState() => _MyPageState();
 }
 
-class _MyAppState extends State<MyApp> {
+class _MyPageState extends State<MyPage> {
+  final _zendesk = Zendesk();
   @override
   void initState() {
     super.initState();
     initZendesk();
   }
 
+  Future<String> getJwtToken() async {
+    return 'JWT_TOKEN';
+  }
+  
   Future<void> initZendesk() async {
     if (!mounted) {
       return;
     }
-             const _accountKey='C4NkxGBoxHSTMW3Gm3gznZ6AxIqYXWkZ';
-  const _appId='ff92947363297c35ad960e50edc747e7a19dbbd7235a852e';
-
-    await Zendesk.initialize(_accountKey, _appId);
+    const _accountKey = 'ACCOUNT_KEY';
+    const _appId = 'APP_ID';
+    await _zendesk.initialize(
+      accountKey: _accountKey,
+      appId: _appId,
+      getJwtToken: getJwtToken,
+    );
   }
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      home: Scaffold(
-        appBar: AppBar(
-          title: const Text('Zendesk Chat Plugin'),
-        ),
-        body: Center(
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              const Padding(
-                padding: EdgeInsets.all(16),
-                child: Text(
-                  'Initialize  example with proper\nkeys in main.dart',
-                  textAlign: TextAlign.center,
-                ),
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Zendesk Chat Plugin'),
+      ),
+      body: Center(
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            const Padding(
+              padding: EdgeInsets.all(16),
+              child: Text(
+                'Initialize  example with proper\nkeys in main.dart',
```

Comment:
```suggestion
                'Initialize example with proper\nkeys in main.dart',
```
acho q sobrou um espaçozinho aqui